content/en/blog/README.md

```diff
@@ -0,0 +1,62 @@
+
+To submit a blog post, follow the steps below.
+
+1. [Sign the Contributor License Agreements](https://github.com/istio/community/blob/master/CONTRIBUTING.md#contributor-license-agreements) if you have not yet done so.
+1. Familiarize yourself with the Markdown format for existing blog posts in the [docs repository](https://github.com/istio/istio.io/tree/master/content/en/blog/2020). 
+1. Write your blog post in a text editor of your choice.
+1. Choose at least one of the following tags for your blogpost: New releases, Announcements, Trainings, Events, Commercial products
+1. (Optional) If you need help with markdown, check out [StakEdit](https://stackedit.io/app#) or read [GitHub's formatting syntax](https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax) for more information. 
+1. Create a new folder in the directory for the corresponding year in the [blog repository](https://github.com/istio/istio.io/tree/master/content/en/blog), and click **Create new file**.
+1. Paste your content into the editor and save it. Name the file in the following way: *folder-for-your-blog/index.md* , but don’t put the date in the file name. The blog reviewers will work with you on the final file name, and the date on which the blog will be published.
+1. When you save the file, GitHub will walk you through the pull request (PR) process.
+1. A reviewer is assigned to all pull requests automatically. The reviewer checks your submission, and works with you on feedback and final details. When the pull request is approved, the blog will be scheduled for publication.
+1. Ping editorial team members on Slack [#docs](https://istio.slack.com/archives/C50V5EATT) channel with a link to your recently created PR. 
+
+### Blog Guidelines
+
+#### Suitable content:
+- **Original content only**
+- Istio new feature releases and project updates
+- Tutorials and demos 
+- Use cases
+- Content that relates vendor or platform to Istio installation and use 
+
+#### Unsuitable Content:
+- Content that is disparaging to Istio project, its functionalities or community
+- Content that does not observe our [code of conduct](https://github.com/istio/community/blob/master/CONTRIBUTING.md#code-of-conduct)
+- Blogs that do not address Istio in any way
+- Content that doesn't interact with Istio APIs or interfaces
+- Blatant vendor pitches
+
+#### Vendor blog posts
+Contributors are encouraged to show how Istio’s functionality can be extended and interesting use cases can be addressed using technology, and it is fine to write about commercial technology in doing so. Each vendor has a limit of once a month to post on the Istio blog. However, it is important that the blog post teaches the reader something they didn’t know about Istio rather than coming across as a commercial for the vendor’s technology. It will be up to the judgement of the editorial lead whether a post is “just an ad” or is truly adding value.
```

Comment:
Is it not clear already by stating that these are _vendor_ blogposts?

Reply:
(I can't re-open the PR sorry.)

I think you can clear this up like so:

#### Commercial blog posts
Contributors are encouraged to show how Istio’s functionality can be extended and interesting use cases can be addressed using third-party technology, and it is fine to write about commercial products in doing so. We allow vendors one post per month discussing such technology on the Istio blog. However, it is important that the blog post teaches the reader something they didn’t know about Istio rather than coming across as a commercial for the vendor’s technology. It will be up to the judgement of the editorial lead whether a post is “just an ad” or is truly adding value.